README.md

```diff
@@ -20,10 +20,18 @@ import '@github/auto-check-element'
 </auto-check>
 ```
 
+```erb
+<auto-check src="/signup-check/username">
+  <input>
+  <div data-csrf="<%= authenticity_token_for("/signup-check/username") %>"></div>
```

Comment:
Not a huge fan of this API myself but actually wanted to avoid using a input element since it will get posted to the backend and could cause issues.

I'm open to suggestion on alternatives.

Reply:
As discussed in #js, I think using an unnamed `input` element would work well here, that would not be submitted either. So:

```
<input type="hidden" class="this-wont-submit-without-a-name" value="foo">
```